### PR TITLE
vaadin version updates

### PR DESCRIPTION
**Pull request by svenruppert**

### CODE REVIEW

**Comment by Ladicek** [comment]
ok to test

**Comment by svenruppert** [comment]
@Ladicek , anything I can do here to get it merged ?

**Comment by Ladicek** [comment]
Hmm, so, I don't know why Jenkins didn't run a build :-/ Let me try again.

test this please

**Comment by svenruppert** [comment]
Hi @ladicek, I can not see the details why it is failing on Windows.

**Comment by Ladicek** [comment]
Yea, we're not sure why the build logs aren't accessible to anonymous viewers, but I took a look and it's totally unrelated to this change, so no worries.
